app/incident/note/delete/template.hbs

```diff
@@ -1,6 +1,6 @@
 {{#modal-dialog isUpdateDisabled=isUpdateDisabled title=title updateButtonAction=updateButtonAction updateButtonText=updateButtonText}}
   <div class="alert alert-danger">
     <span class="glyphicon glyphicon-warning-sign"></span>
-    {{t 'incident.messages.deleteNote'}}
+    {{t 'messages.delete_singular' name='note'}}
```

Comment:
This code is passing a non localized string when it should be passing in a localized string or it should use the name of the item being deleted.